setup.py

```diff
@@ -64,6 +64,8 @@ def run_tests(self):
           'Programming Language :: Python :: 3.6',
           'Programming Language :: Python :: 3.7',
           'Programming Language :: Python :: 3.8',
+          'Programming Language :: Python :: 3.9',
+          'Programming Language :: Python :: 3.10',
```

Comment:
The CI is only covering up to Python 3.9 above. Is that asymmetry intended?

To make Python 3.10 work in the CI, and update from bionic/18.04 to hirsute/21.04 may be needed. 

PS: Python 3.5 and 3.6 support can be dropped by now, as far as I am concerned.

Reply:
I tested those changes on 3.10
Should I add python 3.10 to .travis.yml?

Reply:
My _personal_ vote would be "yes please, that would be welcome". @gauravv7 are there any reasons to stick to bionic for longer?

Reply:
PS: I noticed use of `ppa:deadsnakes/ppa` just now. I imagine that repository _may_ even allow use of python3.10 even with bionic, just to add to the pool of options.

Reply:
Yes, the tox tests were failing at that time, Travis build was failing due to timeout while setting up tox to run tests. So I ran based on bionic.

Reply:
@mikekeda @hartwork we can add 3.10 in tox and Travis configs